view/type/TypePage.kt

```diff
@@ -85,7 +83,7 @@ sealed class TypePage(
 ) : Page() {
 
     override val hasSecondary: Boolean = false
-    override val icon: Form.IconArg = typeIcon(type)
+    override val icon: Form.IconArg = conceptIcon(type.conceptType)
```

Comment:
One consequence of introducing the Concept Preview library is that we do end up looking at the `conceptType` (from Client Java) a lot more here than we used to. I don't know if this is good or not, given that we already have an abstraction for Client Java's `Type`, which is our `TypeState`. However, `TypeState` is very heavyweight (>500 lines) and much of that is tailored to the needs of the Type Browser and the Type Page.